### PR TITLE
Adding an external workshops folder and instructions

### PR DESCRIPTION
**Pull request by Jen-OMalley**
This PR addresses #130. I am adding an `external-workshops` folder to this repo with a README, screenshots, and instructions for using tools and materials.

### CODE REVIEW

File: external-workshops/using-tools-and-materials.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+* Make sure you are the owner of the new repository and click `Create Fork`.
+<img src="screenshots/reproducible-research-fork-2.png" width=800>
+
+* Now you will have your own copy of the repository and can make any modifications to the materials. Be sure to update the `workshop-schedule` and `participant_information` files. These can be found in your copy of the [`docs` folder](https://github.com/AlexsLemonade/reproducible-research/tree/main/docs).
+
+**2.) You will also be using another repository, called [`rrp-workshop-exercises`](https://github.com/AlexsLemonade/rrp-workshop-exercises).**
+This separate repository holds exercises, data, and scripts used during the interactive portion of the course.
+(During the workshop, participants will also fork and clone`rrp-workshop-exercises` to use as an example repository!)
+
+Although you will be walking participants through forking this repository in real-time during the course, you should certainly create your own copy and get familiar with the exercises and materials ahead of time.
```
[the remark this answers is not in the document]
Comment (Jen-OMalley) [reply external-workshops/using-tools-and-materials.md]: Added a similar sentence. Let me know if that works!

[the remark this answers is not in the document]
Comment (sjspielman) [reply external-workshops/using-tools-and-materials.md]: looks good!

File: external-workshops/using-tools-and-materials.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+The [`setup_instructions` folder](https://github.com/AlexsLemonade/reproducible-research/tree/main/docs/setup_instructions) in the `reproducible-research` repository contains helpful information for installing these tools. 
+
+## Training materials
+
+**1.) The [`reproducible-research` repository](https://github.com/AlexsLemonade/reproducible-research) holds materials such as lecture slides, cheatsheets, and instructor notes.**
+Fork the repository, to create your own copy of the materials.
+
+* Navigate to the [`reproducible-research` repository](https://github.com/AlexsLemonade/reproducible-research) and click `Fork`.
+<img src="screenshots/reproducible-research-header-fork-1.png" width=800>
+
```
Comment (sjspielman) [inline external-workshops/using-tools-and-materials.md:14]: Spacing is a little tight here when I actually look at the file, so let's force a new line - 

```suggestion
<br>
```

File: external-workshops/using-tools-and-materials.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+* Now you will have your own copy of the repository and can make any modifications to the materials. 
+* The`docs` directory contains content for the course website.
+	* Be sure to update the `workshop-schedule` and `participant_information` for your own workshop!
+* The `instructor_notes` directory contains notes for live demonstration instruction. Refer to these notes for helpful guidance on teaching certain activities in this course.
+* The `workshop_materials` directory contains additional workshop materials such as cheatsheets and links to the lecture slides.
+
+
+**2.) You will also be using another repository, called [`rrp-workshop-exercises`](https://github.com/AlexsLemonade/rrp-workshop-exercises).**
+This separate repository holds exercises, data, and scripts used during the interactive portion of the course.
+(During the workshop, participants will also fork and clone`rrp-workshop-exercises` to use as an example repository!)
```
Comment (sjspielman) [inline external-workshops/using-tools-and-materials.md:27]:
```suggestion
(During the workshop, participants will also fork and clone `rrp-workshop-exercises` to use as an example repository!)
```

**Review by sjspielman — APPROVED** [review]
LGTM! Left a few small text comments but don't need to see again! Feel free to merge after changes; no need to wait for my PR to get merged first.

**Pull request merged**